README.md

```diff
@@ -19,14 +19,14 @@ This repository is a lerna mono-repo which is home to multiple next-auth adapter
 
 Currently, there are the following adapters ready for use
```

Comment:
Maybe also change the text here until #68 is complete to explicitly recommend / warn to use the canary versions? @balazsorban44

Reply:
Yes, I think we should say something like these are experimental adapters, and we would like feedback on them!

Reply:
Alright, updated! What do you think? 

I tried to make it clear what the plan is regarding the legacy  adapters as well.